<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessAnalysisWidget</class>
 <widget class="QWidget" name="AccessAnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccessAnalysisWidget</string>
  </property>
  <widget class="QLabel" name="startPointLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出发点</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetPointLayer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目标点图层</string>
   </property>
  </widget>
  <widget class="QLabel" name="arcLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>道路表</string>
   </property>
  </widget>
  <widget class="QLabel" name="nodeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>交叉路口表</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLimitLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>时间范围</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeLimitTextLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="arcBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单位：米</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nodeBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="targetPointLayerBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="startBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
